docs/CONOPS_TEMPLATE.adoc

```diff
@@ -0,0 +1,112 @@
+:toc: macro
+
+[cols="1,1,1,1,1,1"]
+|===
+6+^s|Title
+6+^|<title>
+
+1+s|Document Type
+2+|Conops
+1+s|Originating Department
+2+|Systems
+
+s|Document ID
+|<document id>
+s|Status
+|<status>
+s|Created
+|<dd/mm/yyyy>
+
+s|Authored by
+|<author>
+s|Reviewed by
+|<reviewer>
+s|Approved by
+|<approver>
+
+|===
+
+toc::[]
+
+<<<
+
+_The ConOps, at the organization level, addresses the leadership's intended way of operating the organization. It may refer to the use of one or more systems, as black boxes, to forward the organization's goals and objectives. The ConOps document describes the organization's assumptions or intent in regard to an overall operation or series of operations of the business with using the system to be developed, existing systems, and possible future systems. This document is frequently embodied in long-range strategic plans and annual operational plans. The ConOps document serves as a basis for the organization to direct the overall characteristics of the future business and systems, for the project to understand its background, and for the users of this International Standard to implement the stakeholder requirements elicitation. The ConOps document should include the following information items._
+
+== Purpose
+
+_Describe the purpose of this document by describing the current status of the organization, the goal of the long-range strategic plan, and gaps between them._
+
+== Scope
+
+_Describe the scope of this document by specifying the organization's business domains to which it applies._
```

Comment:
@capsulecorplab this Conops seems much more focused on the people running the operation than the engineering and mechanical operations of the payload itself. This is in contrast to [John Paterson's ConOps](https://docs.google.com/document/d/1-hRz5XKsrLemaTI3zKRS6VFRAF2KNrFSMr4wgYlgxso/edit) which is much more focused on the latter.

Should we add another section that's more narrowly focused on the engineering/mechanical processes, or is that meant to be embedded in the bigger context that's outline here?

Reply:
I noticed that too. We can either deviate from the original ISO/IEC/IEEE 29148:2011 spec and rewrite the template to be more engineering focused, or we can try to refactor John's ConOps doc to fit into the bigger context as you suggest. I don't have a strong preference toward either.

Reply:
John had actually written the OpsCon, where as Sean has focused on Conops. They are slightly different. Check this source for a quick difference: https://plus.google.com/104099794938550497171/posts/D51e89gh4RR

Reply:
Document type in this case is Mission Concept and ConOps is the document name

Reply:
the rest of the document seems fine. However, we can start investigating to determine each points mentioned in the table of content